### PR TITLE
Comment Change - Katie Klenda

### LINKED ISSUE #1: Add collaborators to this repo
## Comment below to be added as a collaborator

We will all be using this repository today. Even though it's public right now, you will not be able to make changes until you're given the correct permissions. We've automated this process via the GitHub API. Once you comment, we will add you as a collaborator.

You'll start to receive a lot of emails. (❗) You can slow these down immediately by clicking the **Unwatch** button at the top of the page and selecting either the **Participating and @mentions** or **Ignore** option.

### PR DESCRIPTION
**Pull request by kl-wells**
### Change based around comment for issue. This resolves #1

### CODE REVIEW

File: _slides/07-kl-wells.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
----
-
-Caption here
-
+---
+layout: slide
+title: "Welcome to our slide deck!"
+---
+
+New caption this is :)
```
Comment (kl-wells) [inline _slides/07-kl-wells.md:6]: This looks great! Love the smiley face!

**Review by kl-wells — COMMENTED** [review]
This looks great to me. Approve comment - it's positive.

File: _slides/07-kl-wells.md (hunk 1 above)
Comment (kellMorrGEA) [reply _slides/07-kl-wells.md:6]: very nice caption!

**Review by kellMorrGEA — APPROVED** [review]
looks great, adding my approval

**Pull request merged**